[WIP]

jjscarafia opened the pull request.

jjscarafia committed:
[WIP]

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-academic/pull/93).

jjscarafia committed:
[REF]

jjscarafia committed:
[REF] fix computation

jjscarafia committed:
[ADD] academic: partner link views

jjscarafia committed:
[ADD] academic: relationships menu

jjscarafia committed:
[FIX] academic: translation and views

jjscarafia committed:
[IMP] academic: partner data and views

lef-adhoc committed:
[IMP] academic: add style to handle line break between company_type and partner_type

jjscarafia committed:
[FIX] academic: typo on parents action

lef-adhoc committed:
[FIX] academic: fix error when adding or changing a contact and role

ica-adhoc committed:
[ADD] academic: hide partner fields

fields are hidden base on partner_type

ica-adhoc committed:
[IMP] academic: compute company on partners

company is computed if the partner type is setted

ica-adhoc committed:
[IMP] academic: default parent on family create partner

If create partner on family type form view, it
will create with parent partner type by default

lef-adhoc committed:
[FIX] academic: compute iteration is added

lef-adhoc committed:
[ADD] academic_split_name: add new module

lef-adhoc committed:
[FIX] academic: company adjustment that is not just reading

maq-adhoc committed:
Merge branch '16.0' into 16.0-t-34157-jjs

lef-adhoc committed:
[IMP] academic: warning of contact with the same dni

lef-adhoc committed:
[IMP] academic: when you choose partner that automatically shows you the children
